src/common/getPullRequestCommits.ts

```diff
@@ -1,5 +1,5 @@
 import { getOctokit } from '@actions/github';
-import { GraphQlQueryResponseData } from '@octokit/graphql';
+import type { GraphQlQueryResponseData } from '@octokit/graphql';
```

Comment:
Importing as type prevents the `import/no-extraneous-dependencies` from throwing errors about this dependency being a devDependency